src/util/getTime.test.ts

```diff
@@ -0,0 +1,16 @@
+import getTime from './getTime';
+
+test('should return number from string', () => {
+  const testData = '2019-04-22T21:43:53.000Z';
+  expect(getTime(testData)).toEqual(1555969433000);
+});
+
+test('should return number from Date', () => {
+  const testData: Date = new Date('2019-04-22T21:43:53.000Z');
+  expect(getTime(testData)).toEqual(1555969433000);
+});
+
+test('should return undefined from undefined', () => {
+  const testData = undefined;
+  expect(getTime(testData)).toEqual(undefined);
+});
```

Comment:
We can ditch this code, there is a [`parseTimePropertyValue` function in the new SDK](https://github.com/JupiterOne/sdk/blob/41b5788dae6c6fce4d5f0848dde40f20981cc6cc/packages/integration-sdk-core/src/data/converters.ts#L171).

Reply:
Ditched